collaboration/1.slack.md

```diff
@@ -0,0 +1,101 @@
+# Slack
+
+[Slack](https://slack.com/) is a chat we use to discuss about anything.
```

Comment:
I wanted to provide some context/reference for what Slack is, but if that's distracting I can remove it.

On the other hand, I think people reading this document will already be on Slack and if they are not the link to our slack is not a useful information since they don't have an account.